drivers/charger/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
+add_subdirectory_ifdef(CONFIG_SBS_CHARGER sbs_charger)
```

Comment:
is there any need to use subdirs? Except sensors, other drivers just have one level

Reply:
Yes, but it's not the common pattern.

Reply:
nack, this does not sound like a hard rule and I would like a better reason than "it is because it is" before I go implement this change.

Reply:
Let's turn it around a bit. What is the gain from splitting into subdirs for charges? Most driver classes don't do this.

Reply:
+1, please flatten dir structure unless folders are strictly needed

Reply:
ack, we're spending too much time on getting an answer here. 
I would appreciate if you all would add it to the style guidelines, so new contributors don't get caught up in this.
Also, it was really messy to refactor the commits and I don't have the bandwidth for it, so it will have to be a fourth commit.